projects/stargazer/plugins/other/radius/radius.cpp

```diff
@@ -121,9 +121,38 @@ int RAD_SETTINGS::ParseSettings(const ModuleSettings & s)
         pva = std::find(pvi->sections.begin(), pvi->sections.end(), pv);
         if (pva != pvi->sections.end() && !pva->value.empty())
         {
-            printfd(__FILE__, "ParseSettings Value of match: '%s'\n", pva->value[0].c_str());
+            printfd(__FILE__, "ParseSettings Value of auth/match: '%s'\n", pva->value[0].c_str());
 
-            m_matchPairs = ParseAuthAttr(pva->value[0], pv.param.c_str());
+            m_matchPairs = ParseAuthAutzAttr(pva->value[0], pv.param.c_str());
+
+            for (const auto& at : m_matchPairs)
+                printfd(__FILE__, "Key: '%s', Value: '%s', Type: %d\n", at.first.c_str(), at.second.value.c_str(), at.second.type);
+        }
+    }
+
+    pv.param = "autz";
+    pvi = std::find(s.moduleParams.begin(), s.moduleParams.end(), pv);
+    if (pvi != s.moduleParams.end())
+    {
+        pv.param = "send";
+        auto pva = std::find(pvi->sections.begin(), pvi->sections.end(), pv);
+        if (pva != pvi->sections.end() && !pva->value.empty())
+        {
+            printfd(__FILE__, "ParseSettings Value of autz/send: '%s'\n", pva->value[0].c_str());
+
+            m_sendPairs = ParseAuthAutzAttr(pva->value[0], pv.param);
```

Comment:
Here you overwrite the `m_sendPairs`, which contains the value from the 'auth' section. You need additional pair of `m_sendPairs` and `m_matchPairs` for the 'autz' section.